spec/models/course/assessment/category_spec.rb

```diff
@@ -43,5 +43,32 @@
         expect(subject.folder.name).to eq(new_title)
       end
     end
+
+    context 'when there is another category with the same title' do
+      let(:course) { create(:course) }
+      let(:common_title) { 'Assessments' }
+      let!(:category) { create(:course_assessment_category, course: course, title: common_title) }
+
+      context 'after category was created' do
+        subject { build(:course_assessment_category, title: common_title, course: course) }
+
+        it 'creates a folder with the proper name' do
+          subject.save
+
+          expect(subject.folder.name).to eq(subject.title + '(0)')
```

Comment:
That looks weird and it's hard to define the "correct" behaviour.  Suppose I uploaded a file named `file.txt` which its name is in english, but my language is chinese. Should the file be renamed to `file（０）.txt` ?